widgetssdk/src/main/java/com/glia/widgets/base/BasFadeActivity.kt

```diff
@@ -0,0 +1,34 @@
+package com.glia.widgets.base
+
+import android.content.Intent
+import android.os.Bundle
+import androidx.appcompat.app.AppCompatActivity
+
+open class BasFadeActivity : AppCompatActivity() {
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        overridePendingTransition(android.R.anim.fade_in, android.R.anim.fade_out)
+        super.onCreate(savedInstanceState)
```

Comment:
I think `super` calls should be on the first line of the function everywhere in this class.